### PR TITLE
Update "Game Dashboard" for "Manage Tab".md

### PR DESCRIPTION
**Pull request by LuniMoon**
1st version of the "Game Analytics" page.

### CODE REVIEW

File: docs/gdevelop5/interface/games-dashboard/game-analytics.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+# Game Analytics
+
+
+!!! note
+
+    **Who can use this feature:** All users with a GDevelop account who had published their game on gd.games.
+
+Player analytics are essential during video game development. They provide data on how long players stay engaged with the game and how quickly they lose interest.
+By understanding these patterns, developers can tailor game features to better suit player preferences, enhancing enjoyment and engagement.
+Alongside [player feedback](https://wiki.gdevelop.io/gdevelop5/interface/games-dashboard/player-feedback/), this information guides improvements, helping ensure the game's success by making it more appealing and enjoyable for its audience.
```
Comment (4ian) [inline docs/gdevelop5/interface/games-dashboard/game-analytics.md:14]: For links inside the wiki, remove `https://wiki.gdevelop.io`:
```suggestion
Alongside [player feedback](/gdevelop5/interface/games-dashboard/player-feedback/), this information guides improvements, helping ensure the game's success by making it more appealing and enjoyable for its audience.
```

The first slash (/) in the link indicates that "this is relative to the root of the website".

**Comment by Bouh** [comment]
Is the page aim to replace this one that sill exist?
https://gdevelop-wiki-git-fork-lunimoon-patch-10-gdevelop.vercel.app/gdevelop5/interface/games-dashboard/game-analytics/

**Comment by LuniMoon** [comment]
Urgh. I did something wrong.
I misplaced the "Game Analytics" content.
I've should have updated https://gdevelop-wiki-git-fork-lunimoon-patch-10-gdevelop.vercel.app/gdevelop5/interface/games-dashboard/game-analytics/, with the new content; but I just noticed that I've placed the new content inside the "Game Dashobard" page (https://gdevelop-wiki-git-fork-lunimoon-patch-10-gdevelop.vercel.app/gdevelop5/interface/games-dashboard/) instead of replacing "Game Analytics". -____-

I'll fix this when I come back from vacation.

**Comment by LuniMoon** [comment]
Uh... I'm a  little confused (probably because of my use GitHub and versioning comprehension) but, I see that the Wiki online is correct? 
I see:
* Game Dashboard > Manage (https://gdevelop-wiki-git-fork-lunimoon-patch-10-gdevelop.vercel.app/gdevelop5/interface/games-dashboard/) 
* Player feedback (updated with the latest version) https://wiki.gdevelop.io/gdevelop5/interface/games-dashboard/player-feedback/
* Leaderboards §untouched by me) https://wiki.gdevelop.io/gdevelop5/interface/games-dashboard/leaderboard-administration/
* Marketing (untouched) https://wiki.gdevelop.io/gdevelop5/interface/games-dashboard/marketing/

But the Vercel preview is displaying something else? (https://gdevelop-wiki-git-fork-lunimoon-patch-10-gdevelop.vercel.app/gdevelop5/interface/games-dashboard/) Game Dashboard does not include player feedback and so on...
Anyway, the online version of the Wiki is OK for me. The Vercel preview that I see can be discarded :)

**Pull request closed without merging**